### PR TITLE
Add solid inspired chair election process to draft CG charter (attempt #2)

### PR DESCRIPTION
**Pull request by bumblefudge**
Apologies for the messy PR but the first commit just reformats the markdown as HTML and the second includes (in addition to some messy indent diffs) all the changes I made. Feel free to open comments or change-suggestions reversing my opinionated edits!

### CODE REVIEW

File: CGCharter-1727386911.html [hunk 1]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+    <p>      
+      For most Chair nominees, the primary affiliation is their employer and
+      will match their affiliation in the W3C database. For contractors and
+      independents, this will normally be their contracting company or their
+      independent status; in some cases (e.g. where a consultant is consulting
+      for only one organization) this may be the organization for whom the
+      nominee is consulting.
+      <strong>Chair nominees, elected chairs, and appointed chairs MUST have
+      unique affiliations publicly stated at time of nomination, election (if it
+      has changed since last disclosure), or appointment.</strong>
```
Comment (ThisIsMissEm) [inline CGCharter-1727386911.html:285]: Can an chair nominee explicitly choose to run independent of their employer? i.e., they're aware that their affiliation and running for chair is not supported by their employer, and that they may be changing employers in the near future?

If so, language like "A chair nominee may choose to run independent of their current company, so long as their affiliation with that company changes within 6 months of election, otherwise they will be dismissed from their post and the chair position will automatically hold a new election for that role.

**Review by ThisIsMissEm — COMMENTED** [review]
A few questions / comments.

File: CGCharter-1727386911.html [hunk 2]
```diff
@@ -201,21 +201,26 @@ <h2 id="transparency">
...
+      mailing list</a>.
     </p>
     <h2 id="decision">
       Decision Process
     </h2>
     <p class="remove">
-      If the decision policy is documented somewhere, update this section accordingly to link to it.
+      If the decision policy is documented somewhere, update this section
+      accordingly to link to it.
     </p>
```
[the remark this answers is not in the document]
Comment (bumblefudge) [reply CGCharter-1727386911.html:224]: yeah `class="remove"` would kinda imply that, huh?  🤦‍♀️

File: CGCharter-1727386911.html [hunk 3]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        be up for election. Thereafter, in each year, a single election will be
+        held to fill any vacant seats.
+      </li>
+      <li>
+        In the case of interim vacancy, the remaining chairs may appoint a
+        co-chair for each open seat, hold an election for the same, or wait
+        until the next election, at their discretion. If the chairs do not take
+        any action, the seat will automatically be up for election in the next
+        cycle. Any such interim appointments or elections shall hold the seat
+        until the end of its natural cycle.
```
[the remark this answers is not in the document]
Comment (bumblefudge) [reply CGCharter-1727386911.html:313]: the Solid CG charter that I modeled this on had _appointments_ for the rest of the existing term and _elections_ only at regular times, I believe-- maybe I should make that more explicit here.

File: CGCharter-1727386911.html (hunk 1 above)
Comment (bumblefudge) [reply CGCharter-1727386911.html:285]: I think the idea was that if one of the current chairs has affiliation X, no nominees or appointees should also have that same affiliation.  I would appreciate some kind of flexibility for people self-attesting their affiliation as "independent", but what's more valuable to be is the _recent disclosure_ about, say, current employment, substantial stock holdings, serving on Board, etc., because the group consensus _could well be_ to accept that nominee's/appointee's word that they're acting in an individual capacity, but should _not_ make such exceptions without all the facts fresh at hand.

File: CGCharter-1727386911.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (dariusk) [reply CGCharter-1727386911.html:313]: I had to read the paragraph a few times but did come to @bumblefudge's conclusion: an interim person in a position (whether appointed or elected) lasts until the end of the natural cycle of the seat the person was appointed into. So in @ThisIsMissEm's case, yes, the person being appointed or elected would hold the seat for 18 months instead of 24. Not sure it needs rewording as-is.

File: CGCharter-1727386911.html [hunk 4]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        publicly and on the mailing list.
+      </li>
+      <li>
+        Chairs may also be removed from their duties through a no-confidence
+        vote.
+      </li>
+      <li>          
+        If a participant of the community group wishes to call for the recall of
+        a chair, for any reason, that participant must first privately
+        communicate with one or more of the other chairs and/or W3C contact
```
Comment (dariusk) [inline CGCharter-1727386911.html:374]:
```suggestion
        communicate with one or more of the other chairs and/or W3C Team Contact
```

Suggestion: make this explicit, it's not just a general contact but *the* Team Contact, right?

File: CGCharter-1727386911.html [hunk 5]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        their desire and reason for doing so. Those chairs must give the
+        participant an opportunity to discuss their concerns with a goal of
+        resolving them. If, after 30 days, the participant feels their issues
+        have not been adequately addressed, they may then escalate to a public
+        call for a no confidence vote (without having to disclose with whom they
+        have been discussing the matter). If, after 60 days, the participant has
+        not made a call for a no confidence vote, the matter should be
```
Comment (dariusk) [inline CGCharter-1727386911.html:381]:
```suggestion
        not made a call for a no-confidence vote, the matter should be
```

File: CGCharter-1727386911.html [hunk 6]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+      </li>
+      <li>          
+        If a participant of the community group wishes to call for the recall of
+        a chair, for any reason, that participant must first privately
+        communicate with one or more of the other chairs and/or W3C contact
+        their desire and reason for doing so. Those chairs must give the
+        participant an opportunity to discuss their concerns with a goal of
+        resolving them. If, after 30 days, the participant feels their issues
+        have not been adequately addressed, they may then escalate to a public
+        call for a no confidence vote (without having to disclose with whom they
```
Comment (dariusk) [inline CGCharter-1727386911.html:379]:
```suggestion
        call for a no-confidence vote (without having to disclose with whom they
```

File: CGCharter-1727386911.html [hunk 7]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        have been discussing the matter). If, after 60 days, the participant has
+        not made a call for a no confidence vote, the matter should be
+        considered dropped; any further attempts to remove the chair in question
+        must begin with a new round of private communication.
+      </li>
+      <li>
+        A public call for no confidence must be announced to the group email
```
Comment (dariusk) [inline CGCharter-1727386911.html:386]:
```suggestion
        A public call for no-confidence must be announced to the group email
```

File: CGCharter-1727386911.html [hunk 8]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        list stating the name of the chair subject to the recall. This
+        announcement must come from the participant who initiated the private
+        communication with the chairs to discuss their concerns. At least two
+        other supporting participants MUST reply to that email on the public
+        list within 48 hours to confirm their support for a no-confidence vote
+        of the chair in question.
+      </li>
+      <li>
+        The other chairs must acknowledge the call for no confidence within 7
```
Comment (dariusk) [inline CGCharter-1727386911.html:395]:
```suggestion
        The other chairs must acknowledge the call for no-confidence within 7
```

File: CGCharter-1727386911.html [hunk 9]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        days of all three participants declaring their support.
+      </li>
+      <li>
+        UNLESS the chair in which no-confidence has been expressed voluntarily
+        offboards and opts out of the session on the mailing list, and within 30
+        days of a call for no confidence, the other chairs MUST hold a
```
Comment (dariusk) [inline CGCharter-1727386911.html:401]:
```suggestion
        days of a call for no-confidence, the other chairs MUST hold a
```

File: CGCharter-1727386911.html [hunk 10]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        conference call at which the parties seeking no confidence will have an
```
Comment (dariusk) [inline CGCharter-1727386911.html:402]:
```suggestion
        conference call at which the parties seeking no-confidence will have an
```

File: CGCharter-1727386911.html [hunk 11]
```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
...
+        greater than two-thirds of the total votes cast, then the chair in
+        question is removed and the seat shall be treated as an interim vacancy.
+      </li>
+      <li>
+        Participants are not required to vote. Abstentions may be recorded; such
+        abstentions shall not count towards the total number of votes when
+        calculating the two-thirds majority.
+      </li>
+      <li>          
+        A Chair who has been removed may stand for re-election.
```
Comment (dariusk) [inline CGCharter-1727386911.html:423]:
```suggestion
        A Chair who has been removed may stand for reelection.
```

(normalizing to above)

File: CGCharter-1727386911.html (hunk 1 above)
Comment (dariusk) [reply CGCharter-1727386911.html:285]: I don't like a "6 month" clause or whatever because then that still would give a bad actor 6 months to do chair-ish things while colluding with another chair in the interest of a shared organization (which is the issue we are trying to avoid here, right?). We wouldn't just throw out the 6 months of work done by the CG upon dismissing the institutionally-redundant chair, right? And yet that would be 6 months of "compromised" work.

Basically I think it's okay the way it is, and if someone wants to be Chair that badly and they currently work at the same company as another Chair, they can simply quit that job or resign that affiliation. Or wait until the next election.

File: CGCharter-1727386911.html (hunk 4 above)
Comment (csarven) [reply CGCharter-1727386911.html:374]: W3C Team Contact is indeed more specific than a general W3C contact. However, unlike W3C Groups, CGs do not have an assigned W3C Team Contact in the same way. Similarly, CG charters are not formally recognised or enforced by W3C. There is no specific "contact" assigned unless a special arrangement is made between a CG and the W3C Team to designate one.

That said, CGs primarily communicate with the W3C Team's Community Process team at [team-community-process@w3.org](mailto:team-community-process@w3.org) for general admin or process-related matters and potentially for initial Code of Conduct issues. This doesn't extend to CG charters or other CG-specific processes.

So:

```suggestion
        communicate with one or more of the other chairs
```

Aside: The Solid CG charter mentions "W3C Team Contact" in one place but that is technically inaccurate/inapplicable.

**Comment by dmitrizagidulin** [comment]
Thank you everyone for the fix suggestions! Merging the draft, as always feel free to open further PRs.

File: CGCharter-1727386911.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (bumblefudge) [reply CGCharter-1727386911.html:313]: i didn't wanna make it too explicit and binding but my thought at the time was, "not every vacancy justifies the hassle of an election, if it's just to plug a gap and avoid an election, consensus of all chairs to replace someone stepping down with someone else that has similar skills and expertise should be good enough, and one of the other chairs would probably requests/demand an election any time it's not that low-stakes.  happy for anyone to reword to be clearer, although i'd recommend we not make the appointment option too rule-bound; if anything, i would just make it more explicit that all chairs have to explicitly approve any appointment and that, like any other decision, CFC should give entire group a chance to object, request clarification, etc.
